storybook/webpack.config.ts

```diff
@@ -151,10 +151,25 @@ export default ({ config }) => {
   // Required for the storysource storybook addon
   config.module.rules.push(storybookSource)
 
+  config.module.rules.push({
+    test: /^((?!\.stories).)*\.tsx?$/, // match all .tsx files other than .stories.tsx files
+    include: resolve(__dirname, "../packages/component-library"),
+    use: [
+      require.resolve("ts-loader"),
+      require.resolve("react-docgen-typescript-loader"),
+    ],
+  })
+
   config.module.rules.push(
     ...[babel, styles, svgs, svgIcons, elm].map(excludeExternalModules)
   )
 
+  config.module.rules[0].use[0].options.plugins = [
+    require.resolve("babel-plugin-react-docgen"),
```

Comment:
It wasn't clear if you needed to add both `react-docgen` _and_ `typescript-docgen`. I didn't try it without `react-docgen`